meetings/azure-solutions-architect/azure-solutions-architect-qa-2020-11-26.md

```diff
@@ -20,12 +22,22 @@ Elementy który warto wziąć pod uwagę to:
 - Cosmos DB
 - Azure AD
 - Private Endpoint (dla BARDZO chętnych)
+
 ## Linki
+
+### Ogólne linki architektoniczne
 - Cloud Design Patterns: https://docs.microsoft.com/en-us/azure/architecture/patterns/
+- Webinar z Chmurowiska z trzema przykładami architektur: https://bit.ly/39x8qKG
```

Comment:
Może lepiej https://docs.microsoft.com/en-us/azure/architecture/browse/? Jest też sporo lepeij na channel9 to tego odniesień

Reply:
bosko, nie znałam tego linku 😄

Reply:
zmienione